packages/contracts-bedrock/src/dispute/lib/Types.sol

```diff
@@ -40,12 +40,15 @@ library GameTypes {
     /// @dev A dispute game type the uses the cannon vm.
     GameType internal constant CANNON = GameType.wrap(0);
 
-    /// @dev A permissioned dispute game type the uses the cannon vm.
+    /// @dev A permissioned dispute game type that uses the cannon vm.
     GameType internal constant PERMISSIONED_CANNON = GameType.wrap(1);
 
-    /// @notice A dispute game type the uses the asterisc VM
+    /// @notice A dispute game type that uses the asterisc VM
     GameType internal constant ASTERISC = GameType.wrap(2);
 
+    /// @notice A dispute game type that uses OP Succinct
+    GameType internal constant OP_SUCCINCT = GameType.wrap(3);
```

Comment:
Game type 3 is actually asterisc-kona: https://github.com/ethereum-optimism/optimism/blob/43b4e77c8b206c2518a2748e0b8e36bb73eab1bd/op-challenger/game/fault/types/types.go#L30

We're also adding a new game type for interop which is game type 4 currently (and probably 5 for permissioned game).

This makes me wonder if we actually need to add game types here. Game types are meant to be a generic plugin mechanism so I wonder if we actually need/want these enums in the code.  The deploy script would probably still set constants for the game types it deploys but they don't seem to need to be more general than that given asterisc-kona is working.

What do you think @clabby?

Reply:
If we don't add all the game types here we probably should add OP_SUCCINT to that list in op-challenger just to reserve a game type code as life is a lot simpler if we don't reuse the same code for different things.

Reply:
Yeah this is a bit out of sync.

I have a preference for keeping this enum in the smart contracts mainly for the deploy scripts + testing, but we need some way to keep it in sync.

For now, would you mind adding `ASTERISC_KONA` as game type == `3`, and using `4` for yours @leruaa?

Reply:
@clabby double checking, isn't `4` for `SuperCannonGameType`?

Reply:
@leruaa It is - apologies, didn't rtfd. Let's go with 5, and if you could also add in `ASTERISC_KONA` @ `3` we're good to go :)

Reply:
@clabby done. I also moved `OP_SUCCINCT` to `6` as @ajsutton said `5` will probably be taken.